### PR TITLE
[PS-4837] Chnage way how we provide my.cnf

### PR DESCRIPTION
**Pull request by EvgeniyPatlan**

### CODE REVIEW

File: build-ps/debian/percona-server-common-8.0.postinst [hunk 1]
```diff
@@ -2,75 +2,10 @@
...
-  ;;  
-
-  *)  
-    echo "postinst called with unknown argument '$1'" 1>&2
-    exit 1
-  ;;  
-esac
+if [ "$1" = "configure" ]; then
+	# Low priority fallback for client use when no server is installed.
+	echo "!includedir /etc/mysql/conf.d/" >> /etc/mysql/my.cnf.fallback
```
Comment (EvgeniyPatlan) [inline build-ps/debian/percona-server-common-8.0.postinst]: this is needed as if we copy all upstream behavior we get
` trying to overwrite '/etc/mysql/my.cnf.fallback', which is also in package mysql-common 5.8+1.0.4
dpkg-deb: error: paste subprocess was killed by signal (Broken pipe)
`

**Review by abychko — APPROVED** [review]
lgtm

File: build-ps/debian/percona-server-server-8.0.install [hunk 2]
```diff
@@ -16,6 +16,11 @@ usr/bin/resolveip
 usr/bin/resolve_stack_dump
 usr/sbin/mysqld
 usr/bin/ps-admin
+
+# confguration files
+debian/extra/mysql.cnf  etc/mysql/
```
Comment (RoelVdP) [inline build-ps/debian/percona-server-server-8.0.install:21]: etc or /etc ??

File: build-ps/debian/percona-server-server-8.0.install [hunk 3]
```diff
@@ -16,6 +16,11 @@ usr/bin/resolveip
...
+debian/extra/mysqld.cnf /etc/mysql/mysql.conf.d/
```
Comment (RoelVdP) [inline build-ps/debian/percona-server-server-8.0.install:22]: here it is /etc

Comment (EvgeniyPatlan) [reply build-ps/debian/percona-server-server-8.0.install:22]: it should be etc/. anyway I'm reworking this part right now

File: build-ps/debian/percona-server-server-8.0.install [hunk 4]
```diff
@@ -16,6 +16,11 @@ usr/bin/resolveip
 usr/bin/resolve_stack_dump
 usr/sbin/mysqld
 usr/bin/ps-admin
+
+# confguration files
```
Comment (laurynas-biveinis) [inline build-ps/debian/percona-server-server-8.0.install:20]: "configuration"

Comment (EvgeniyPatlan) [reply build-ps/debian/percona-server-server-8.0.install:20]: will fix in next commit

**Pull request merged**